### PR TITLE
Update azure.appservice.easyauth.jar and azure.appservice.jar

### PR DESCRIPTION
**Pull request by chunye**
Update azure.appservice.easyauth.jar to https://msazure.visualstudio.com/One/_git/AAPT-Antares-EasyAuth-Extensions-Java/commit/7b6c769980dc3b457f276bbc03e5507ab6cb5737?refName=refs%2Fheads%2Fdev.

Update azure.appservice.jar to https://msazure.visualstudio.com/One/_git/AAPT-Antares-Websites-Extensions-Java/commit/d655ad58dc5fa7cdbc8ee9863a6161da39cd154b?refName=refs%2Fheads%2Fdev.

Removed unused reference and artificats of javax.json*.

### CODE REVIEW

File: shared/tomcat/8.5/catalina.properties [hunk 1]
```diff
@@ -108,7 +108,6 @@ shared.loader=
 tomcat.util.scan.StandardJarScanFilter.jarsToSkip=\
 azure.appservice.jar, \
 azure.appservice.easyauth.jar, \
-javax.json-*, \
```
Comment (jvano) [inline shared/tomcat/8.5/catalina.properties:111]: why is this no longer needed?

File: shared/tomcat/9.0/catalina.properties [hunk 2]
```diff
@@ -108,7 +108,6 @@ shared.loader=
 tomcat.util.scan.StandardJarScanFilter.jarsToSkip=\
 azure.appservice.jar, \
 azure.appservice.easyauth.jar, \
-javax.json-*, \
```
Comment (jvano) [inline shared/tomcat/9.0/catalina.properties:111]: +1

**Review by jvano — CHANGES_REQUESTED** [review]
:clock1:

File: shared/tomcat/8.5/catalina.properties (hunk 1 above)
Comment (shrishrirang) [reply shared/tomcat/8.5/catalina.properties:111]: https://github.com/Azure-App-Service/tomcat/blob/dev/shared/easyauth/javax.json-1.0.4.jar and https://github.com/Azure-App-Service/tomcat/blob/dev/shared/easyauth/javax.json-api-1.0.jar are not used anymore. @chunye: would be good to delete them.

Comment (shrishrirang) [reply shared/tomcat/8.5/catalina.properties:111]: Use of javax.json* was removed by https://github.com/Azure-App-Service/tomcat/pull/22/files#diff-19a3ddf241387c5fa3007123bb153484

Comment (chunye) [reply shared/tomcat/8.5/catalina.properties:111]: These are deleted in this PR.

File: shared/tomcat/9.0/catalina.properties (hunk 2 above)
Comment (chunye) [reply shared/tomcat/9.0/catalina.properties:111]: Same reason for 8.5 applies to 9.0.

File: shared/tomcat/8.5/context.xml [hunk 3]
```diff
@@ -25,7 +25,5 @@
     <WatchedResource>/home/site/deployments/active</WatchedResource>
 
     <!-- Uncomment this to disable session persistence across Tomcat restarts -->
-    <!--
     <Manager pathname="" />
-    -->
 </Context>
```
Comment (shrishrirang) [inline shared/tomcat/8.5/context.xml:28]: Missed 9.0/context.xml?

Comment (chunye) [reply shared/tomcat/8.5/context.xml:28]: You are right.  Fixed now

**Comment by shrishrirang** [comment]
Looks good to me

**Pull request merged**